astroquery/query.py

```diff
@@ -342,6 +345,9 @@ def _download_file(self, url, local_filepath, timeout=None, auth=None,
         response.close()
         return response
 
+    def __ftp_download(self, url, local_filepath):
```

Comment:
note you used dunder (__) here, but single-underscore on line 253 above.   I think single-underscore is better.

Reply:
Oh! That's a mistake one my part! It was not visible to me on my screen. Sorry for that! :(